resources/l10n/ko/LC_MESSAGES/loot.po

```diff
@@ -1,18 +1,18 @@
 msgid ""
 msgstr ""
-"Project-Id-Version: LOOT v0.7\n"
-"Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2015-01-09 10:40-0000\n"
-"PO-Revision-Date: 2015-01-18 13:26+0900\n"
+"Project-Id-Version: LOOT 0.16.0\n"
```

Comment:
That's a problem with the template, and the version isn't so accurate either, since this translation is for something between 0.16.1 and the next release.